src/Widgets/DataLabelWidget/DataLabelWidgetClass.ts

```diff
@@ -5,7 +5,7 @@ export const DataLabelWidget = provideWidgetClass('DataLabelWidget', {
     label: 'string',
     attributeName: 'string',
     details: 'string',
-    showAs: ['enum', { values: ['text', 'currency', 'datetime'] }],
+    showAs: ['enum', { values: ['text', 'currency', 'datetime', 'url'] }],
```

Comment:
* [ ] FYI It's not just rendered as a URL but as a link with the URL as linked text. Maybe the more important "show-as" aspect is `link` not `url`?